src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -150,11 +151,15 @@ class ChatSwitcherView extends React.Component {
             })
             .map((report) => {
                 const participants = lodashGet(report, 'participants', []);
+                const reportPersonalDetails = getPersonalDetailsForLogins(participants, this.props.personalDetails);
+                const hasMultipleParticipants = participants.length > 1;
+                const lastActorDetails = reportPersonalDetails.find(({login}) => login === report.lastActorEmail);
                 const isSingleUserDM = participants.length === 1;
                 const login = isSingleUserDM ? report.participants[0] : '';
                 return {
                     text: report.reportName,
-                    alternateText: isSingleUserDM ? login : report.reportName,
+                    lastMessageText: (hasMultipleParticipants && lastActorDetails
```

Comment:
Sorry I should have been more clear in this case. We can just focus on the main sidebar options for now. This code will all be removed very soon. So all changes here can be undone and there is no need to export `getPersonalDetailsForLogins()`.